publication-commons/src/main/java/no/unit/nva/publication/model/business/logentry/PublicationLogEntry.java

```diff
@@ -0,0 +1,63 @@
+package no.unit.nva.publication.model.business.logentry;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.time.Instant;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.publication.service.impl.ResourceService;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public record PublicationLogEntry(SortableIdentifier identifier, SortableIdentifier resourceIdentifier, LogTopic topic,
+                                  Instant timestamp, LogUser performedBy) implements LogEntry {
+
+    public static final String TYPE = "PublicationLogEntry";
```

Comment:
Public?

Reply:
Yes, it used in LogEntry interface as subtype property.